src/ima.js

```diff
@@ -55,7 +55,7 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
    */
   static defaultConfig: Object = {
     debug: false,
-    delayInitUntilSourceSelected: false,
+    delayInitUntilSourceSelected: true,
```

Comment:
should be true for ios only

Reply:
Added in _setDefaultConfiguration method